src/Draft.js

```diff
@@ -43,7 +42,6 @@ const DraftPublic = {
   EditorState,
 
   CompositeDecorator: CompositeDraftDecorator,
-  Entity: DraftEntity,
   EntityInstance: DraftEntityInstance,
```

Comment:
With the new changes I've made I think we can remove the export and the `addEntity` method. (If a consumer is not ever required to add an entity manually like that, which I think they shouldn't be)